tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestCLDRLocaleCoverage.java

```diff
@@ -97,7 +97,11 @@ public void TestLanguageNameCoverage() {
         LanguageTagParser ltp = new LanguageTagParser();
         for (String locale : factory.getAvailableLanguages()) {
             String language = ltp.set(locale).getLanguage();
-            if (language.equals("root")) language = "und";
+            if (language.equals("root")) {
+                language = "und";
+            } else if(!StandardCodes.isLocaleAtLeastBasic(language)) {
+                continue;
```

Comment:
this prevents one failure, `regularPlus.containsAll(mainLocales)`

problem is, it not only skips locales that were in seed (like "aa"), but also locales that were in common, like "agq" -- consequently it results in a new failure, `localesForNames.containsAll(coverageLocales)`